ios/LICENSE

```diff
@@ -0,0 +1,219 @@
+Copyright 2013 Appcelerator, Inc.
```

Comment:
Why not use the [existing license](https://github.com/viezel/NappDrawer/blob/master/LICENSE) in the top-level directory? The copyright is also not at appc but Mads.

Reply:
You absolutely right, I missed the top level license 🤦🏻‍♂️